OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -78,7 +78,7 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 				return NextActivity;
 
 			var type = Target.Type;
-			if (!Target.IsValidFor(self) || type == TargetType.FrozenActor)
+			if (!Target.IsValidFor(self))
 				return NextActivity;
 
 			if (attack.Info.AttackRequiresEnteringCell && !positionable.CanEnterCell(Target.Actor.Location, null, false))
```

Comment:
I haven't tested it, but I expect this will crash because `Target.Actor` is null for `TargetType.FrozenActor`.  The rest of this method will need to be updated to deal with the frozen actor's `ActorInfo`.